### PR TITLE
Updated contract for query and distinct

### PR DESCRIPTION
**Pull request by faraonc**

### CODE REVIEW

File: lib/document.proto [hunk 1]
```diff
@@ -2,6 +2,45 @@ syntax="proto3";
...
 package hwsc;
 
+message Publisher {
+    // @inject_tag: bson:"lastName"
+    string last_name = 1;
+    // @inject_tag: bson:"firstName"
+    string first_name = 2;
+}
+
+message Region {
```
Comment (faraonc) [inline lib/document.proto]: StudySite

File: int/hwsc-document-svc/proto/hwsc-document-svc.proto [hunk 2]
```diff
@@ -30,12 +32,14 @@ message DocumentRequest {
     string url = 2;
     string fuid = 3;
     FileType media = 4;
```
Comment (faraonc) [inline int/hwsc-document-svc/proto/hwsc-document-svc.proto]:
```
    string url = 2;
    string fuid = 3;
    FileType media = 4;
```

we should define a new message in document.proto
```
message FileMetadataTransaction{
    string url = 2;
    string fuid = 3;
    FileType media = 4;
}
```

File: lib/document.proto [hunk 3]
```diff
@@ -2,6 +2,45 @@ syntax="proto3";
...
+    // @inject_tag: bson:"callTypes"
+    repeated string call_types = 3;
+    // @inject_tag: bson:"groundTypes"
+    repeated string ground_types = 4;
+    // @inject_tag: bson:"sensorTypes"
+    repeated string sensor_types = 5;
+    // @inject_tag: bson:"sensorNames"
+    repeated string sensor_names = 6;
+    // @inject_tag: bson:"minSampleRate"
+    uint32 min_sample_rate = 7;
```
Comment (faraonc) [inline lib/document.proto]: del

File: lib/document.proto [hunk 4]
```diff
@@ -2,6 +2,45 @@ syntax="proto3";
...
+    // @inject_tag: bson:"maxSampleRate;"
+    uint32 max_sample_rate = 8;
```
Comment (faraonc) [inline lib/document.proto]: del

File: lib/document.proto (hunk 3 above)
Comment (kimlisa) [inline lib/document.proto]: we don't need lines 34 - 37 anymore (min/max sample rate)
Kerri said we don't need users querying sample rates

**Pull request merged**